src/main/java/lesson9Text/TextAnalyzer.java

```diff
@@ -0,0 +1,23 @@
+package lesson9Text;
+
+public class TextAnalyzer {
```

Comment:
Ta logika powinna wyglądać trochę inaczej. Według zadania potrzebujemy tylko jednej metody która będzie:
1. **Sprawdzać, czy podany text nie jest pusty, a jeśli tak to wyświetl**i informację o tym czy jest on palindromem, czy nie.
2. **Sprawdzać, czy podany text nie jest pusty, a jeśli tak to wyświetl** długość tego tekstu.

Reply:
Zawiłe, czy to znaczy, że ma sprawdzać 2 razy czy tekst nie jest pusty? Chyba mija się to z celem?

Reply:
Sam program jest jedną metodą, która używa innych odpowiedzialnych za kolejne kroki algorytmu. Dodawanie kolejnych metod było również sugerowane przez Ciebie w komentarzu niżej odnośnie odwracania tekstu.

Reply:
poprawione